backend/apps/owasp/api/search/project.py

```diff
@@ -30,7 +31,20 @@ def get_projects(query, attributes=None, limit=25):
         "typoTolerance": "min",
     }
 
-    return raw_search(Project, query, params)["hits"]
+    project_cache_key = f"algolia_search_{query}"
+    project_cache_result = cache.get(project_cache_key)
+
+    # check if cache exists
+
+    if project_cache_result:
+        return project_cache_result
+
+    search_result = raw_search(Project, query, params)["hits"]
+
+    # save to cache
+
+    cache.set(project_cache_key, search_result, 86400)
```

Comment:
Could you create a separate constant for `86400` value in the constants.py file?